Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/question.ts

```diff
@@ -0,0 +1,13 @@
+﻿import { Question } from './question.model';
+import { SingleMultipleQuestion } from './single-multiple-question';
+import { CodeSnippetAC } from './code-snippet';
+
+export class QuestionBase {
+    question: Question;
+    singleMultipleAnswerQuestion: SingleMultipleQuestion;
+    codeSnippetQuestion: CodeSnippetAC = null;
```

Comment:
initialization should go in a constructor. Why did you require to initialize it with null?